### DISCUSSION TITLE
**TYPO**  in config causing invalid character being displayed for Sunday on saver page

### DISCUSSION THREAD

**Discussion by BigBobbas** [BigBobbas/ESP32-S3-Box3-Custom-ESPHome#22] [Announcements]
Hi Everyone, 
I have just noticed a typo in the date and month That causes an invalid character prefixing the words Sunday and January. there are 2 x { that shouldn't be there. they are shown below and below that, the corrected yaml.
![image](https://github.com/BigBobbas/ESP32-S3-Box3-Custom-ESPHome/assets/150487209/cbc61a69-4f69-494a-a50d-8b037de62089)

correct below
```yaml
      - id: saver_page
        lambda: |-
         it.fill(Color::BLACK);
         
         const char *days[] {"${sunday}","${monday}","${tuesday}","${wednesday}","${thursday}","${friday}","${saturday}"};
         it.printf(165, 10, id(my_font3), light_blue, TextAlign::RIGHT, days[id(ha_time).now().day_of_week - 1]);
         
         it.strftime(180, 10, id(my_font3), light_blue, "%d", id(ha_time).now());
         
         const char *months[] {"${jan}","${feb}","${mar}","${apr}","${may}","${jun}","${jul}","${aug}","${sept}","${oct}","${nov}","${dec}"};
         it.printf(220, 10, id(my_font3), light_blue, TextAlign::LEFT, months[id(ha_time).now().month - 1]);
         
         
         it.strftime(160, 85, id(my_fontseg2), TextAlign::CENTER, "%H :%M", id(ha_time).now());
         it.printf(160, 180, id(my_fontseg2), light_blue, TextAlign::CENTER, "%.f°C", id(s3temp).state);
```

Thank you for putting up with the little glitches. We will get all the creases ironed out asap. Please raise an issue if you are experiencing problems with functions not working or not working as expected.  and for ideas, feedback any any offers of help to make this even better! then please open a discussion in the relevant topic. 
Thanks for your support and for joining in with testing.
Cheers 
Rob

**Comment by Bluhme1** [comment]
I didn’t see this PR before I answered @youkorr in the other thread.
Glad to help in making this awesome firmware more perfect.
/Lars

**Comment by youkorr** [comment]
perfect this firmware
